docker/README.md

```diff
@@ -62,12 +62,90 @@ docker-compose -f docker-compose-mysql.yml down
 
 Please note that SQL support is still in active developement and it is not production ready yet.
 
-Quickstart for production
+One host production setup with docker-compose
 =========================
-In a typical production setting, dependencies (cassandra / statsd server) are
-managed / started independently of the cadence-server. To use the container in
-a production setting, use the following command:
+For a simple production setup, start with our **docker-compose-production-example.yml** for a single host Cadence cluster as described in the following steps.
 
+#### 1. Modify the database mounting path in the example YAML file
+You may want to set the Cassandra data location because:
+* a. You need a disk partition that has enough space
+* b. You prefer to have separated storage hardware for have a separated storage hardware for database for performance, backup plan, and availability (with RAID)
+* c. Setting a mounting path can avoid losing data when you run **docker-compose down**
+
+Save the edited example YAML file as docker-compose.yaml.
+
+See the [Docker Documentation](https://docs.docker.com/compose/production/) for more information about using docker-compose in production.
+
+#### 2. Run docker-compose
+```
+docker-compose up -d
+```
+Try out our [samples](https://github.com/uber-common/cadence-samples) to make sure the server is running well.
+
+#### 3. Upgrade Cadence server
+* a. First, you need to upgrade the database schema. You can find instructions for using the cadence-cassandra-tool to upgrade schema [here](https://github.com/uber/cadence/tree/master/tools/cassandra#update-schema-as-part-of-a-release).
+* b. Shut down docker-compose by using the following command:
```

Comment:
We can't do in-place upgrades?

Reply:
Ideally yes, we should be able to do that. But with docker-compse, you have to stop and then start so that it will pick up the new docker image.
I have tried "docker-compose restart", it simply restart the container without reading the changes from compose file.